docs/hyn/5.2/installation.md

```diff
@@ -49,6 +49,7 @@ Under connections:
 ```
 
 Most often it makes sense to use either `system` or `tenant` as your default database connection setting; this is `DB_CONNECTION` in your `.env` file. This environment variable is assigned to the `default` key in the `config/database.php` configuration file.
+When setting the default database to system or tenant you do not need to use Laravel default `mysql` anymore. You can use for `mysql` and `system` the same settings, but it doesn't have to be. 
```

Comment:
Could you undo these changes, they have been discussed at length today here: https://github.com/tenancy/docs/pull/48

Reply:
For me this was the most challenging part to understand your concept. Is anything wrong on the statement? Otherwise I would prefer to keep it in as could be helpful for people like me which just get started.

Reply:
Your sentence still leaves too much open for interpretation:

 "You can use for `mysql` and `system` the same settings, but it doesn't have to be."

This could be interpreted in so many ways; stating "it doesn't have to be" only makes it harder to decide. I'd say stick to the previously improved sentence.

Thanks for jumping onto it, I understand the suggestion and there's much we can improve. If we get more of these suggestions I'll look into clarifying that full flow; I'm hoping the tutorial I'm writing will clarify some of the tenant/system choices as well. See: 
https://github.com/tenancy/docs/blob/dk/landing-tutorial/docs/hyn/5.2/tutorial-with-landing.md

Reply:
However, done